### PR TITLE
Cleanup all and enable pylama

### PR DESCRIPTION
**Pull request by mirceaulinic**
<!-- Make sure you have read http://napalm.readthedocs.io/en/latest/contributing/index.html --!>

### CODE REVIEW

**Comment by mirceaulinic** [comment]
@napalm-automation/council please review

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ install:
 - pip install -r requirements.txt
 - pip install .
 - pip install -e git+https://github.com/napalm-automation/napalm-base.git@develop#egg=napalm-base
+- pip install pylama
```
Comment (dbarrosop) [inline .travis.yml]: Add a requirements-dev.txt and install from there

**Review by dbarrosop — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (mirceaulinic) [reply .travis.yml]: Updated

**Review by dbarrosop — APPROVED** [review]
+1

**Pull request merged**